clang/lib/Driver/Driver.cpp

```diff
@@ -979,6 +1000,45 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
     return;
   }
 
+  //
+  // SYCL
+  //
+  // We need to generate a SYCL toolchain if the user specified -fsycl.
+  // If -fsycl is supplied without any of these we will assume SPIR-V.
```

Comment:
nit: not clear what 'any of these' refers to.
Could be re-phrased to something like: `If -fsycl-targets is not specified, then single SPIR-V target is assumed`

Reply:
This initial PR will not introduce `-fsycl-targets`, we will just rely on `-fsycl` to imply `spir64` for JIT.  I will clean up the comments.